src/main/java/de/uni_tuebingen/ub/nppm/db/TinyMCE_ContentDB.java

```diff
@@ -1,7 +1,7 @@
 package de.uni_tuebingen.ub.nppm.db;
 
 import de.uni_tuebingen.ub.nppm.model.*;
-
+import de.uni_tuebingen.ub.nppm.model.TinyMCE_Content.Context;
```

Comment:
Noch eine "Kleinigkeit": Wenn die Tabelle von TinyMCE_Content umbenannt wurde in content, dann sollte auch die DB und Model Klasse sowie das Servlet umbenannt werden.

Reply:
mach ich gleich